react-toggle-switch-component/my-app/src/App.css

```diff
@@ -0,0 +1,82 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.slider {
+  width: 4rem;
+  height: 1.5rem;
+  border: 2px solid black;
+  border-radius: 1rem;
+  background-color: white;
+  display: flex;
+  margin: 1rem;
+  transition: ease 1s;
+}
+
+button.toggle {
+  padding: 0.75rem;
+  border: 0;
+  border-radius: 50%;
+  height: 90%;
+  transition: ease 1s;
+  cursor: pointer;
+}
+
+.slider.OFF {
+  background-color: white;
+}
+
+.slider.ON {
+  background-color: green;
+}
+
+.toggle.OFF {
+  transform: translateX(0);
+  background-color: rgba(211, 211, 211, 0.692);
+}
+
+.toggle.ON {
+  transform: translateX(2.5rem);
+  background-color: rgba(0, 175, 0, 0.822);
+}
```

Comment:
These should be in a seperate css file `ToggleSwitch.css` and import that file into your `ToggleSwitch` component. Since these styles are only being applied to your `ToggleSwitch` component, it's better to just import these there.